samples/graph-activity-feed/nodejs/views/configure.ejs

```diff
@@ -54,6 +54,8 @@
         // see an error indicating that the configuration settings could not be saved.
         saveEvent.notifySuccess();
     });
+  });
```

Comment:
Please run build pipeline for this PR and add the link

Reply:
Yess build pipeline is successfull